src/components/Switch/switch.component.css

```diff
@@ -0,0 +1,51 @@
+.inputWrapper {
+    position: relative;
+    top: 5px;
+    left: 5px;
+}
+
+.input {
+    position: absolute;
+    left: -9999px;
+    top: -99999px;
+}
+
+.slider {
+    display: flex;
+    cursor: pointer;
+    width: 35px;
+    height: 15px;
+    border-radius: 100px;
+    background-color: #C6C6C6;

```

Comment:
Can you please use a custom property for all colours? https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

e.g `background-color: var(--grey-500);`